src/widgets/conditions/conditions-detailed-summary.component.test.tsx

```diff
@@ -0,0 +1,194 @@
+import React from "react";
+import { performPatientConditionSearch } from "./conditions.resource";
+import { render, cleanup, wait } from "@testing-library/react";
+import { BrowserRouter, match } from "react-router-dom";
+import ConditionsDetailedSummary from "./conditions-detailed-summary.component";
+import { useCurrentPatient } from "../../../__mocks__/openmrs-esm-api.mock";
+
+const mockPerformPatientConditionSearch = performPatientConditionSearch as jest.Mock;
+const mockUseCurrentPatient = useCurrentPatient as jest.Mock;
+
+jest.mock("./conditions.resource", () => ({
+  performPatientConditionSearch: jest.fn()
+}));
+
+jest.mock("@openmrs/esm-api", () => ({
+  useCurrentPatient: jest.fn()
+}));
+
+const mockPatientConditionResult = {
```

Comment:
Could you move this mocked data into the __mocks__ directory?

Reply:
Ok

Reply:
Yes